yacc.go

```diff
@@ -3470,22 +3469,20 @@ func $$Parse($$lex $$Lexer) int {
 
 func ($$rcvr *$$ParserImpl) Parse($$lex $$Lexer) int {
 	var $$n int
-	var $$lval $$SymType
```

Comment:
> `rcvr.stack` is allocated with the `rcvr`, not a separate allocation. Ditto for `rcvr.lval`

Wasn't this variable previously on stack, though? And if it's a pointer, isn't it still being allocated separately?